radix-engine-interface/src/types/node_layout.rs

```diff
@@ -6,40 +6,40 @@ use sbor::rust::prelude::*;
 // Please update REP-60 after updating types/configs defined in this file!
 //=========================================================================
 
-pub const TYPE_INFO_BASE_MODULE: ModuleNumber = ModuleNumber(0u8);
-pub const METADATA_BASE_MODULE: ModuleNumber = ModuleNumber(1u8);
-pub const ROYALTY_BASE_MODULE: ModuleNumber = ModuleNumber(2u8);
-pub const ACCESS_RULES_BASE_MODULE: ModuleNumber = ModuleNumber(3u8);
-pub const OBJECT_BASE_MODULE: ModuleNumber = ModuleNumber(32u8);
+pub const TYPE_INFO_BASE_PARTITION: PartitionNumber = PartitionNumber(0u8);
```

Comment:
What does `BASE` mean here? The start of the partition counts? But I thought as per REP-60 these partitions were fixed by the system?

Perhaps the following might be more accurate?
* `TYPE_INFO_FIELD_PARTITION`
* `METADATA_KVSTORE_PARTITION`
* `ROYALTY_FIELD_PARTITION`
* `ACCESS_RULES_FIELD_PARTITION`
* `MAIN_FIELD_PARTITION`
* `MAIN_ADDITIONAL_PARTITIONS_START_FROM`

I see in this PR you've basically used "FIELD" instead of "TUPLE" to describe these partitions - which I much prefer (to save confusion with the SBOR tuple value kind) - would you be up for changing REP-60 to use the name "FIELD" instead of "TUPLE" for this partition type?